spec/consensus/evidence.md

```diff
@@ -171,8 +180,8 @@ message Evidence {
 ```
 
 `DuplicateVoteEvidence` and `LightClientAttackEvidence` are self-contained in
-the sense that the evidence can be used to derive the `abci.Evidence` that is
-sent to the application. Because of this, extra fields are necessary:
+the sense that the evidence can be used to derive the array of `abci.Misbehavior` for each byzantine validator that is
```

Comment:
I really don't understand the point of "self-contained" here. In particular because just after we say that we need more information to provide a `Misbehavior` to the application.

Reply:
re-worded, removed the self-contained sentence, made it more direct